### PR TITLE
Fixed navbar and added 'GLUG MVIT' link in the footer

### PR DESCRIPTION
**Pull request by shreyaraj13**

### CODE REVIEW

File: src/Components/Footer.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ class Footer extends Component {
            </ul> */}
 
            <ul className="copyright">
-              <li>&copy; Copyright 2020 GLUG MVIT</li>
+              <li>&copy; Copyright 2020 <a href="https://blog.glugmvit.com/" target="_blank" >GLUG MVIT</a> </li>
```
Comment (aa-deet-eeya) [inline src/Components/Footer.js]: Add `rel="noopener noreferrer"` to the anchor tag, using `target="_blank"` without those is a serious security vulnerability.
Read more here at  https://mathiasbynens.github.io/rel-noopener/

This is what is causing the builds to fail

**Review by aa-deet-eeya — CHANGES_REQUESTED** [review]

File: src/Components/Footer.js (hunk 1 above)
Comment (shreyaraj13) [reply src/Components/Footer.js]: Thank you for letting me know.

**Comment by aa-deet-eeya** [comment]
lgtm, @cseas what do you think?

**Comment by cseas** [comment]
> lgtm, @cseas what do you think?

There's space for improvements:
- Removing margin/padding from only `h1` in global CSS will make for inconsistent styling across headers since that's the only header without a default margin now and we don't want it to feel alone like that. If you want to override the default CSS, just add a local/inline CSS for h1 in your component. The default style will get overridden, that's how CSS works, more specific styles take preference.
- Creating an id selector just for adding one CSS line is really unnecessary, you could just add that one line CSS inline and save the extra id.
- The code doesn't look formatted, but that's partly our fault because we haven't added any instruction or check for it. I'll add a CI check for prettier formatting to fix this.
- I find the "CSS in JS" method of styling much easier to read and implement because then you don't need to worry about selectors at all. Maybe we can slowly try to encourage that as a good practice.

It's a beginner-friendly project though and we're on a short deadline so none of my above comments really matter too much. They're just suggestions. We can merge anything that seems to do what's intended in the deploy preview and doesn't break/worsen the site.

**Comment by shreyaraj13** [comment]
@cseas Thank you so much for helping me improve my code. I'll work on it. :)

**Pull request merged**